graphene_federation/inaccessible.py

```diff
@@ -0,0 +1,54 @@
+from typing import Callable, Any, Optional
+
+from graphene import Schema, Field
+from graphene.types.schema import TypeMap
+
+
+def get_inaccessible_types(schema: Schema) -> dict[str, Any]:
+    """
+    Find all the inaccessible types from the schema.
+    They can be easily distinguished from the other type as
+    the `@inaccessible` decorator adds a `_inaccessible` attribute to them.
+    """
+    inaccessible_types = {}
+    for type_name, type_ in schema.graphql_schema.type_map.items():
+        if not hasattr(type_, "graphene_type"):
+            continue
+        if getattr(type_.graphene_type, "_inaccessible", False):
+            inaccessible_types[type_name] = type_.graphene_type
+    return inaccessible_types
+
+
+def inaccessible(field: Optional[Any] = None) -> Any:
+    """
+    Decorator to use to inaccessible a given type.
+    """
+
+    # noinspection PyProtectedMember,PyPep8Naming
+    def decorator(Type):
```

Comment:
```suggestion
    def decorator(type_):
```

@erikwrede we use this “convention” in Strawberry for built-ins, shall we do the same here?

Reply:
sounds good, reduces IDE warnings a lot and it's also consistent with core `graphene` 👍